client/script.js

```diff
@@ -5,10 +5,15 @@ const port = '3000'
 const submit = document.getElementById('submit')
 
 
+
 submit.addEventListener('click', (e) => post(e))
 
 window.addEventListener('hashchange', updateContent);
```

Comment:
I like the use of the 'hashchange' event listener I hadn't come across this one prior to this project. Good stuff